Lib/ntpath.py

```diff
@@ -313,7 +312,7 @@ def ismount(path):
 
     if _getvolumepathname:
         x = path.rstrip(seps)
-        y =_getvolumepathname(path).rstrip(seps)
+        y = _getvolumepathname(path).rstrip(seps)
```

Comment:
While I agree that the code is ugly as written, the change here distracts from the primary purpose of this PR and makes the diff harder to review. If merged, it would lead to a confusing git history. It doesn't belong in this PR; please revert this change.